src/lib/coda_base/transaction_logic.ml

```diff
@@ -753,6 +765,15 @@ module Make (L : Ledger_intf) : S with type ledger := L.t = struct
                           (User_command_status.Failure.describe failure)))
             else ret
           in
+          let%map () =
```

Comment:
These need corresponding updates in `sparse_ledger.ml`'s `apply_user_command_exn`.